apps/meeting/src/containers/MeetingForm/index.tsx

```diff
@@ -169,6 +171,7 @@ const MeetingForm: React.FC = () => {
             setMeetingMode(MeetingMode.Attendee);
           } else {
             setMeetingMode(MeetingMode.Spectator);
+            setIsWebAudioEnabled(false);
```

Comment:
can you jsut check if isWebAudioEnabled first and then toggleWebAudio if needed

Reply:
Yep, it reduces the changes, so I went ahead a changed it to that.